src/clib/lib/enkf/obs_vector.cpp

```diff
@@ -53,6 +54,21 @@ struct obs_vector_struct {
     std::vector<int> step_list;
 };
 
+static int find_within_tolerance(std::vector<time_t> time_map, time_t external_time) {
+    int nearest_index = -1;
+    double nearest_diff = 9999999.9;
```

Comment:
Use infinity to show intent. Having just some arbitrary number of nines might convey the same, but it is silly.

```suggestion
    double nearest_diff = INFINITY;
```

or the more C++ (and therefore more wordy):
```suggestion
    auto nearest_diff = std::numeric_limits<double>::infinity();
```
(Might need to `#include <limits>`)